SF iOS/SF iOS/Feed/EventsFeedViewController.m

```diff
@@ -281,7 +282,7 @@ - (void)shareApp {
 
 - (void)viewDidAppear:(BOOL)animated {
     [super viewDidAppear:animated];
-    [self.tableView reloadData];
```

Comment:
It seems this reloadData call is not necessary 🧹 Please let me know if anybody knows a specific reason to call this method here 🙂